app/cinema/hall.py

```diff
@@ -0,0 +1,28 @@
+from app.cinema import bar
+from app.people import customer

```

Comment:
Change these imports to
```suggestion
from app.cinema.bar import CinemaBar
from app.people.customer import Customer

```